Return-Path: <pazke@donpac.ru>
Received: from localhost (bix [127.0.0.1])
	by localhost.localdomain (8.12.10/8.12.10) with ESMTP id i68D1ehL009241
	for <akpm@localhost>; Thu, 8 Jul 2004 06:01:42 -0700
Received: from bix [127.0.0.1]
	by localhost with POP3 (fetchmail-6.2.0)
	for akpm@localhost (single-drop); Thu, 08 Jul 2004 06:01:42 -0700 (PDT)
Received: from fire-2.osdl.org (fire.osdl.org [65.172.181.4])
	by mail.osdl.org (8.11.6/8.11.6) with ESMTP id i68D1dG22460
	for <akpm@mail.gateway.osdl.net>; Thu, 8 Jul 2004 06:01:39 -0700
Received: from donpac.ru (mail.donpac.ru [80.254.111.2])
	by fire-2.osdl.org (8.12.8/8.12.8) with ESMTP id i68D1aws017963
	(version=TLSv1/SSLv3 cipher=EDH-RSA-DES-CBC3-SHA bits=168 verify=NO)
	for <akpm@osdl.org>; Thu, 8 Jul 2004 06:01:38 -0700
Received: from [80.254.111.9] (helo=pazke)
	by donpac.ru with smtp (Exim 4.34)
	id 1BiYWq-0005oM-Jz; Thu, 08 Jul 2004 17:01:35 +0400
Subject: [PATCH 5/5] 2.6.7-mm6, CRC16 renaming in VIA Velocity ethernet driver
In-Reply-To: <10892916902912@donpac.ru>
X-Mailer: gregkh_patchbomb_levon_offspring
Date: Thu, 8 Jul 2004 17:01:33 +0400
Message-Id: <10892916931944@donpac.ru>
Mime-Version: 1.0
Content-Type: text/plain; charset="iso-8859-1"
To: Andrew Morton <akpm@osdl.org>, linux-kernel@vger.kernel.org
Content-Transfer-Encoding: 8bit
From: Andrey Panin <pazke@donpac.ru>
X-MIMEDefang-Filter: osdl$Revision: 1.1.1.1 $
X-Scanned-By: MIMEDefang 2.36
X-Spam-Level: 
X-Spam-Checker-Version: SpamAssassin 2.60 (1.212-2003-09-23-exp) on bix
X-Spam-Status: No, hits=-4.2 required=1.0 tests=BAYES_00,RATWARE_HASH_2_V2 
	autolearn=no version=2.60


Signed-off-by: Andrey Panin <pazke@donpac.ru>

 drivers/net/via-velocity.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff -urpNX /usr/share/dontdiff linux-2.6.7-mm6.vanilla/drivers/net/via-velocity.c linux-2.6.7-mm6/drivers/net/via-velocity.c
--- linux-2.6.7-mm6.vanilla/drivers/net/via-velocity.c	Wed Jul  7 20:07:11 2004
+++ linux-2.6.7-mm6/drivers/net/via-velocity.c	Wed Jul  7 20:36:19 2004
@@ -78,7 +78,7 @@
 #include <linux/ip.h>
 #include <linux/tcp.h>
 #include <linux/udp.h>
-#include <linux/crc16.h>
+#include <linux/crc-ccitt.h>
 #include <linux/crc32.h>
 
 #include "via-velocity.h"
@@ -3086,7 +3086,7 @@ u16 wol_calc_crc(int size, u8 * pattern,
 				continue;
 			}
 			mask >>= 1;
-			crc = crc16(crc, &(pattern[i * 8 + j]), 1);
+			crc = crc_ccitt(crc, &(pattern[i * 8 + j]), 1);
 		}
 	}
 	/*	Finally, invert the result once to get the correct data */
